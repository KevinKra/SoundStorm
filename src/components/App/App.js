import React, { Component, Fragment } from "react";
import { Route } from "react-router-dom";
import AsideNav from "../../containers/AsideNav/AsideNav";
import "./App.css";
import HomePage from "../HomePage/HomePage";
import MainPage from "../MainPage/MainPage";
import ArtistPage from "../ArtistPage/ArtistPage";

class App extends Component {
  render() {
    return (
      <Fragment>
        <Route path="/" component={AsideNav} />
        <Route exact path="/" component={HomePage} />
        <Route path="/browse" component={MainPage} />
        <Route path="/artist" component={ArtistPage} />
      </Fragment>
    );
  }
}

export default App;
