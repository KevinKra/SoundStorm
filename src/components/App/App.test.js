import React from "react";
import ReactDOM from "react-dom";
import App from "./App";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("<App />", () => {
  let wrapper = shallow(<App />);
  it("should match the snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
