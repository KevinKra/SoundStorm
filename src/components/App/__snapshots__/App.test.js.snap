// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should match the snapshot 1`] = `
<Fragment>
  <Route
    component={[Function]}
    path="/"
  />
  <Route
    component={[Function]}
    path="/"
  />
</Fragment>
`;
